backend/wayland/wl_seat.c

```diff
@@ -78,10 +79,19 @@ static void pointer_handle_motion(void *data, struct wl_pointer *wl_pointer,
 		return;
 	}
 
+	uint32_t time_frac_nsec;
+	if (backend->input_timestamps.manager &&
+		backend->input_timestamps.p.tv_sec == time / 1000) {
```

Comment:
Style: one more indent to tell the difference between the condition and the body